src/NuGet.Indexing/ExpandAcronymsFilter.cs

```diff
@@ -0,0 +1,117 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Lucene.Net.Analysis;
+using Lucene.Net.Analysis.Tokenattributes;
+
+namespace NuGet.Indexing
+{
+    public sealed class ExpandAcronymsFilter : TokenFilter
```

Comment:
we don't really need sealed classes. The general guidance is to avoid sealing in all our code